packages/dev/core/src/ShadersWGSL/ShadersInclude/screenSpaceRayTrace.fx

```diff
@@ -9,6 +9,13 @@ fn distanceSquared(a: vec2f, b: vec2f) -> f32 {
     return dot(temp, temp); 
 }
 
+#ifdef SSRAYTRACE_SCREENSPACE_DEPTH
+fn linearizeDepth(depth: f32, near: f32, far: f32) -> f32 {
+  var z: f32 = depth * 2.0f - 1.0f; // Convert [0, 1] range to [-1, 1]

```

Comment:
You should not convert to [-1,1], NDC Z coordinate in WebGPU is between 0 and 1, which is already the range of `depth`.

Reply:
Quite right. The depth should already be in screenspace, which doesn't require the [-1, 1] conversion in either WebGL or WebGPU. I just copied this code from another place and didn't think about it enough. 
I've fixed it now.